modular_skyrat/modules/imported_vendors/code/vendor_snacks.dm

```diff
@@ -246,30 +266,30 @@
 	name = "tizirian dumplings"
 	desc = "A three pack of tizirian style dumplings, not actually stuffed with anything."
 	icon_state = "dumpling"
-	trash_type = /obj/item/trash/vendor_trash/lizard_boxed
+	trash_type = /obj/item/trash/vendor_trash/lizard_box
 	food_reagents = list(/datum/reagent/consumable/nutriment = 3)
 	tastes = list("potato" = 1, "earthy heat" = 1)
 	foodtypes = VEGETABLES | NUTS
 	custom_price = PAYCHECK_LOWER * 1.8
 
-/obj/item/food/vendor_snacks/lizard_boxed/sweetroll
+/obj/item/food/vendor_snacks/lizard_boxed/sweet_roll
```

Comment:
Just make sure you stick to one thing everywhere. lizard_box, lizard_bagged, or lizard_boxed, but not the three at once. Same goes for all the other `x_bagged`.